test/unit/default_appearance_spec.js

```diff
@@ -50,4 +52,148 @@ describe("Default appearance", function () {
       });
     });
   });
+
+  describe("parseAppearanceStream", () => {
+    it("should parse a FreeText (from Acrobat) appearance", () => {
+      const appearance = `
+      0 w
+      46.5 621.0552 156.389 18.969 re
+      n
+      q
+      1 0 0 1 0 0 cm
+      46.5 621.0552 156.389 18.969 re
+      W
+      n
+      0 g
+      1 w
+      BT
+      /Helv 14 Tf
+      0.419998 0.850006 0.160004 rg
+      46.5 626.77 Td
+      (Hello ) Tj
+      35.793 0 Td
+      (World ) Tj
+      40.448 0 Td
+      (from ) Tj
+      31.89 0 Td
+      (Acrobat) Tj
+      ET
+      Q`;
+      const result = {
+        fontSize: 14,
+        fontName: "Helv",
+        fontColor: new Uint8ClampedArray([107, 217, 41]),
+      };
+      expect(parseAppearanceStream(new StringStream(appearance))).toEqual(
+        result
+      );
```

Comment:
With the comment above implemented, we can also ensure that the stream is being reset as intended:
```suggestion
      );
      expect(appearance.pos).toEqual(0);
```